### DISCUSSION TITLE
Is there an expected date yet for Release v1.0.0?

### DISCUSSION THREAD

**Discussion by jekramer** [JanusGraph/janusgraph#3265] [General]
I noticed that manual db-cache invalidation is marked for the 1.0.0 milestone. I'm excited to work with it, and wondering how soon it might be available in a release

**Comment by porunov** [comment]
We don't have a specific date set for 1.0.0 version. Usually we have major releases once per year and multiple minor releases between each major release. JanusGraph 1.0.0 isn't ready yet as there is still work to be done. That said, it should block you from using anything which is already merged in master branch.
Each JanusGraph commit comes with the full distribution bundle. In case you want to have JanusGraph packages available via Maven it's quite easy to fork JanusGraph and add packages publishing to GitHub Maven repository (let me know if you need help here). 
Thus, if you need to use manual db-cache invalidation, you can just use the latest JanusGraph build and / or publish Java  packages.
As a side note, there is also distributed db-cache invalidation on mutation in the process and should be shipped with JanusGraph 1.0.0.

P.S. I don't provide any expectations because we don't have clear expectations yet.
P.P.S. In case you want to use transaction log for manual db-cache invalidation then you might be interested in the following PR as well: #3264
